packages/core/src/domain/report/reportObservable.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { isChromium } from '../../tools/browserDetection'
+import type { Subscription } from '../../tools/observable'
+import { stubReportingObserver, stubCspEventListener } from '../../../test/specHelper'
+import { initReportObservable, CustomReportType } from './reportObservable'
+
+describe(`report observable`, () => {
+  let reportingObserverStub: { reset(): void; raiseReport(type: string): void }
+  let cspEventListenerStub: { dispatchEvent(): void }
+  let consoleSubscription: Subscription
+  let notifyReport: jasmine.Spy
+
+  beforeEach(() => {
+    if (!isChromium()) {
+      pending('no ReportingObserver support')
+    }
+
+    reportingObserverStub = stubReportingObserver()
+    cspEventListenerStub = stubCspEventListener()
+    notifyReport = jasmine.createSpy('notifyReport')
+  })
+
+  afterEach(() => {
+    reportingObserverStub.reset()
+    consoleSubscription.unsubscribe()
+  })
+  ;[CustomReportType.deprecation, CustomReportType.deprecation].forEach((type) => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  })
  
  ;[CustomReportType.deprecation, CustomReportType.deprecation].forEach((type) => {
```